website/docs/reference/configs-and-properties.md

```diff
@@ -52,10 +52,10 @@ In dbt, properties are declared in `.yml` files, in the same directory as your r
 
 | Resource  | Default directory         | Defined by                   |
 |-----------|---------------------------|------------------------------|
-| models    | `models/`                 | [source-paths](source-paths) |
-| sources   | `models/`                 | [source-paths](source-paths) |
-| exposures | `models/`                 | [source-paths](source-paths) |
-| seeds     | `data/` or `models/`      | [data-paths](data-paths)     |
+| models    | `models/`                 | [model-paths](model-paths) |
+| sources   | `models/`                 | [model-paths](model-paths) |
+| exposures | `models/`                 | [model-paths](model-paths) |
+| seeds     | `seeds/` or `models/`      | [seed-paths](seed-paths)     |
 | snapshots | `snapshots/` or `models/` | [snapshot-paths](snapshot-paths) |
 | analyses  | `analyses/` or `models/`  | [analysis-paths](analysis-paths) |
 | macros    | `macros/` or `models/`    | [macro-paths](macro-paths) |
```

Comment:
hmm - can we update these?
- sources and exposures can be defined in any resource path
- I don't think `models/` will keep working for some of these (and it was there for backwards compatibility)

Reply:
Thoughts on just removing this table and leaving the surrounding text?  Since the resource names now line up with the default directories it seems overly verbose?

```
## Where can I define properties?

In dbt, properties are declared in `.yml` files, in the same directory as your resources.

You can name these files `whatever_you_want.yml` and nest them arbitrarily deeply in subfolders within each directory.

We highly recommend that you define properties in dedicated paths alongside the resources they're describing.```